test_tracetools_launch/test/test_tracetools_launch/test_trace_action.py

```diff
@@ -51,32 +55,37 @@ def tearDown(self) -> None:
         if 'LD_PRELOAD' in os.environ:
             del os.environ['LD_PRELOAD']
 
-    def _assert_launch(self, actions) -> int:
+    def _assert_launch(self, actions) -> Tuple[int, LaunchContext]:
```

Comment:
Is there a reason you don't have a type for actions/file in any of these methods?

Reply:
No specific reason, the type annotations are simply incomplete. I can add it here since I'm modifying this line/file anyway